cloud-azure/src/main/java/com/sequenceiq/cloudbreak/cloud/azure/connector/resource/AzureComputeResourceService.java

```diff
@@ -59,10 +58,16 @@
 
     public List<CloudResourceStatus> deleteComputeResources(AuthenticatedContext ac, CloudStack stack, List<CloudResource> cloudResources,
             List<CloudResource> networkResources) {
-        CloudContext cloudContext = ac.getCloudContext();
-        ResourceBuilderContext context = contextBuilder.contextInit(cloudContext, ac, stack.getNetwork(), null, true);
+        ResourceBuilderContext context = initContext(ac, stack);
         context.addNetworkResources(networkResources);
 
         return computeResourceService.deleteResources(context, ac, cloudResources, false);
     }
+
+    private AzureContext initContext(AuthenticatedContext ac, CloudStack cloudStack) {
+        AzureContext context = contextBuilder.contextInit(ac.getCloudContext(), ac, cloudStack.getNetwork(), null, true);
+        context.putParameter(PlatformParametersConsts.RESOURCE_CRN_PARAMETER,
+                cloudStack.getParameters().getOrDefault(PlatformParametersConsts.RESOURCE_CRN_PARAMETER, ""));
```

Comment:
We should at least log if RESOURCE_CRN_PARAMETER is not set in the stack, that should never happen, right?